bindings/python/src/types.rs

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use chrono::Duration;
```

Comment:
import should be grouped below

Reply:
Like this : use chrono::{NaiveDate, NaiveDateTime, Duration};?

Reply:
Done